Felienne/hedy
# [BUG FIX] Join class pre-join page works again

**Description**
In this PR we fix an issue where the pre-join of a class didn't work due to an old `get_page_title` reference. A feature that we deprecated last week. We replace the page title variable with the corresponding YAML key and it's fixed.

**Fixes**
This PR fixes #2234

**How to test**
Navigate to the pre-join page of any class. Either through the share-able link or by invitation. Verify that the page doesn't return an error.
